lib/ally.rb

```diff
@@ -0,0 +1,43 @@
+require 'date'
+require 'oauth'
+
+module Ally
+
+  class Client
+
+    ##
+    # Creates a new Ally client
+    def initialize(client_id, secret, base_url=nil)
+      @client_id = client_id
+      @secret = secret
+      @base_url = base_url || "prod.ally.ac"
+    end
+
+    ##
+    # Sign a request to the Ally API
```

Comment:
Can you indicate this will "Create a signed request to the Ally API", the way it is right now implies it should take in a request.